web/src/components/Navigation/TopNav.tsx

```diff
@@ -116,42 +123,44 @@ export default function TopNav({ searchValue, setSearchValue }: Props) {
                 className={"w-11/12 max-w-xl sm:w-2/3"}
               />
               <LinkIconButton
-                href={"/create"}
+                href={`/create?prompt=${searchValue}`}
```

Comment:
Same question about url encoding, I think we need that here unless there is next.js magic